packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -0,0 +1,126 @@
+import { useSelector } from 'utils/reducer'
+
+import { AccountListItem } from './AccountListItem'
+import { sharedMessages } from './sharedMessages'
+import { AccountsYouManagePageProps, AccountsYouManagePages } from './types'
+const { getUserId } = accountSelectors
+
+const messages = {
+  takeControl:
+    'Take control of your managed accounts by making changes to their profiles, preferences, and content.',
+  noAccounts: 'You don’t manage any accounts.',
+  somethingWentWrong: 'Something went wrong. Please try again later.'
+}
+
+export const AccountsYouManageHomePage = ({
+  setPage
+}: AccountsYouManagePageProps) => {
+  const userId = useSelector(getUserId) as number
+  const { data: managedAccounts, status } = useGetManagedAccounts({ userId })
+  const [approveManagedAccount, approveResult] = useApproveManagedAccount()
+  const [rejectManagedAccount, rejectResult] = useRemoveManager()
+  const { toast } = useContext(ToastContext)
+
+  const handleStopManaging = useCallback(
+    ({ userId }: { userId: number; managerUserId: number }) => {
+      setPage(AccountsYouManagePages.STOP_MANAGING, { user_id: userId })
+    },
+    [setPage]
+  )
+
+  const handleApprove = useCallback(
+    ({
+      currentUserId,
+      grantorUser
+    }: {
+      currentUserId: number
+      grantorUser: UserMetadata
+    }) => {
+      approveManagedAccount({ userId: currentUserId, grantorUser })
+    },
+    [approveManagedAccount]
+  )
+
+  const handleReject = useCallback(
+    ({
+      currentUserId,
+      grantorUser
+    }: {
+      currentUserId: number
+      grantorUser: UserMetadata
+    }) => {
+      rejectManagedAccount({
+        userId: grantorUser.user_id,
+        managerUserId: currentUserId
+      })
+    },
+    [rejectManagedAccount]
+  )
+
+  useEffect(() => {
+    if (
+      approveResult.status === Status.ERROR ||
+      rejectResult.status === Status.ERROR
+    ) {
+      toast(messages.somethingWentWrong)
+    }
+  }, [toast, approveResult.status, rejectResult.status])
+
+  return (
+    <Flex direction='column' gap='xl'>
+      <Text variant='body' size='l'>
+        {messages.takeControl}{' '}
+        <TextLink href='#' variant='visible'>
+          {sharedMessages.learnMore}
+        </TextLink>
+      </Text>
+      {status !== Status.SUCCESS ? (
+        <Box pv='2xl'>
+          <LoadingSpinner
+            css={({ spacing }) => ({
+              width: spacing['3xl'],
+              margin: '0 auto'
+            })}
+          />
+        </Box>
+      ) : null}
+      {status === Status.SUCCESS &&
+      (!managedAccounts || managedAccounts.length === 0) ? (
+        <>
+          <Divider />
+          <Box>
+            <Text variant='body' size='l'>
+              {messages.noAccounts}
+            </Text>
+          </Box>
+        </>
+      ) : null}
+      {managedAccounts?.map((m) => {
+        return (
+          <AccountListItem
+            key={m.user.user_id}
+            user={m.user}
```

Comment:
Is there a world where managed accounts is just an array of ideas and each item can select the user?

Reply:
yeah we can defs do that at least as far as this component is concerned (API returns a full user which is nice because we can cache that with Audius-query)

Reply:
The api could be updated to just return the IDs while also caching the user.
Ultimately the reason the API returns full users is so components that immediately need to render them don't have to do a subsequent fetch.
But I'm in favor of not passing full users around if we don't have to.